week1/project/ecommerce/src/components/products.jsx

```diff
@@ -0,0 +1,29 @@
+import './products.css';
+
+const Product = (props) => {
+  return (
+    <li key={props.id} className='product--item'>
+      <div className='product'>
+        <img className='product--image' src={props.image} alt={props.description} />
+        <span className='product--title' title={props.title}>{props.title}</span>
+      </div>
+    </li>
+  )
+}
+
+const ProductsList = (props) => {
+  const {products, state} = props;
+
+  const filter = state.itsEver ? '' : state.pickCategory
```

Comment:
naming is critical .. I think this variable should be categoryName or something similar.
The first time I read the variable name I thought it's filtered items, which confused me
✏️need to modify✏️